localisation/00_wc_economy.csv

```diff
@@ -0,0 +1,19 @@
+#CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
+borrow_money;Borrow 300�;;;;;;;;;;;;x
+borrow_money_desc;You can borrow 300� from some rich ruler or moneylenders.;;;;;;;;;;;;x
+repay_money;Repay 360� Loan;;;;;;;;;;;;;x
+repay_money_desc;You repay the gold you owe.;;;;;;;;;;;;;x
+debt_360_gold;Owes 360�;;;;;;;;;;;;;x
+interest_payment_360_gold;Receives Interest Payments;;;;;;;;;;;;;x
+EVTDESC_WCECO_100;You ask your servants to look for moneylenders who can borrow gold for your goals.\n\nWithin the day, they return with news.;;;;;;;;;;;;x
+EVTOPTA_WCECO_100;[target_moneylender.GetFirstName] you say... deliver [target_moneylender.GetHerHim] my letter!;;;;;;;;;;;;x
+EVTOPTB_WCECO_100;The local community is giving loans... interesting...;;;;;;;;;;;;x
+EVTDESC_WCECO_110;[Root.PrimaryTitle.GetFOA],\nlegends are told about your wealth. May you allow me to borrow a part of your gold. We will return all the gold to the last coin and more.\n\nSigned,\n[target_borrower.GetTitledName];;;;;;;;;;;;x
+EVTDESC_WCECO_115;[Root.PrimaryTitle.GetFOA],\nit will be my honor to be your sponsor. You can pay off your debt whenever you want.\n\nSigned,\n[target_moneylender.GetTitledName];;;;;;;;;;;;x
+EVTDESC_WCECO_120;[Root.PrimaryTitle.GetFOA],\nto my great regret, I am not able to lend you money.\n\nSigned,\n[target_moneylender.GetTitledName];;;;;;;;;;;;x
+EVTDESC_WCECO_150;It has been a long time since you heard the news from [target_moneylender.GetTitledFirstName].\n\nYou have no idea what happened, but looks like at this point we can consider this transaction to be failed.;;;;;;;;;;;;x
+EVTOPTC_WCECO_110;No! Don't bother me anymore!;;;;;;;;;;;;x
+no_borrowing_letters_next_10_years_tooltip;You won't receive any borrowing letter in the next 10 years;;;;;;;;;;;;x
+ask_for_loan;Borrow 300�;;;;;;;;;;;;x
+ask_for_loan_desc;You can try to borrow 300� from �Y[From.GetBestName]�!.;;;;;;;;;;;;x
```

Comment:
Decision target is ROOT, not FROM

Reply:
Fixed in 233a6e51d8931fe5261393d59c42040c11dfbc52.